code/modules/jobs/job_types/roguetown/clinic/clinicdoctor.dm

```diff
@@ -0,0 +1,63 @@
+/datum/job/roguetown/clinicdoctor
+	title = "Clinic Physician"
+	tutorial = "You are a newly funded sawbones for a small upcoming clinic in town, weather you got tired of traveling and wanted to settle down somewhere or your good natured outlook drove you to be closer to the ill and infirmed here you are, funded by a private benefactor and a reasonable two weeks pay provided in advance. and here you stand yet to prove you are worth the coins payed."
```

Comment:
```suggestion
	tutorial = "You are a newly funded sawbones for a small upcoming clinic in town, whether you got tired of traveling and wanted to settle down somewhere or your good-natured outlook drove you to be closer to the ill and infirmed here you are, funded by a private benefactor and a reasonable two weeks pay provided in advance. and here you stand yet to prove you are worth the coins payed."
```
Githubs spellchecker doesn't like "infirmed", but I think the spellchecker is just wrong here.

Reply:
Github has a spell checker!?